### ISSUE TITLE
Text Cursor Forcefully Moves to End with Curly Braces in Nodes on comfyUI Portable Version

### ISSUE THREAD

**Issue by crankynuts** [comfyanonymous/ComfyUI#3073]
After the recent update, there's a bug in all nodes capable of outputting text strings that include curly braces '{}' for wildcards, where the text cursor is forcibly moved to the end of the text. Despite repeated clicking, the cursor reverts back to the end every 0.5 to 1 second, making it impossible to edit text within the node. I've searched Google but couldn't find anyone with a similar issue, which is why I'm registering it here. Currently, as a workaround, I'm editing the text in a note node and then pasting it into the wildcard node, which is incredibly frustrating. Please help!

*FYI, I'm using the portable version of comfyUI.

Thank you.

**Comment by frankyifei** [comment]
I had this same problem.

**Comment by urstrulybala** [comment]
Facing the same issue!

**Comment by ssuukk** [comment]
Me too!

**Comment by cjps-br** [comment]
the problem does not only occur with comfyUI Portable Version. I'm having the same problem and I use the version installed as per the instructions.

**Comment by Malaclypse235** [comment]
Issue confirmed still happening. I fully updated ComfyUI and all custom nodes, and this bug started immediately when I added {brown|black} to one of the the positive prompts.

**Comment by frankyifei** [comment]
is anyone else not using this feature or they are using wildcard node instead

**Comment by RamonGuthrie** [comment]
Yes it is happening to me too, it has been happening about a month for me, I don't know why more people are not talking about this it's a problem

But I have a bigger more annoying problem, which I might have to create a separate issue for but pre-update ComfyUI I had the frustrating Text Cursor Forcefully Moves to End when using {}
![Screenshot 2024-04-19 153307_edit](https://github.com/comfyanonymous/ComfyUI/assets/156688666/63f43882-7dc3-4663-b711-59665365a44e)
Post update this week {} Text stings don't work at all and I get this problem which breaks my output Text and images
![Screenshot 2024-04-19 155457_edit](https://github.com/comfyanonymous/ComfyUI/assets/156688666/592a4e0d-16da-4d17-b826-e5fc7a9558ab)
